src/Microsoft.Azure.SignalR.Management/Configuration/ServiceManagerContext.cs

```diff
@@ -22,6 +22,8 @@ internal class ServiceManagerContext
 
         public bool DisposeServiceProvider { get; set; } = false;
 
+        public NegotiationResponseOptions NegotiationResponseOptions { get; set; } = new NegotiationResponseOptions();
```

Comment:
context is the inner converted options, it should not have the default value if it is converted from the customer faced `ServiceManagerOptions`, @Y-Sindo shall we consider adding some coding guidance in the comment here?